packages/foreman/foreman/foreman.spec

```diff
@@ -315,35 +311,34 @@ Summary: Foreman asset pipeline support
 Group: Applications/System
 Requires: %{name} = %{version}-%{release}
 Requires: nodejs >= 14
+Requires: /usr/bin/npx
```

Comment:
Node to self: this is https://github.com/theforeman/foreman/blob/89c610451c9ea22eb426826e405518e04d68972f/lib/tasks/plugin_assets.rake#L55